run example failed

I ran examples/createPod.sh but it failed.
```
$ mvn -v
Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-18T02:33:14+08:00)
Maven home: /opt/apache-maven-3.5.4
Java version: 1.8.0_181, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk1.8.0_181.jdk/Contents/Home/jre
Default locale: en_CN, platform encoding: UTF-8
OS name: "mac os x", version: "10.13.6", arch: "x86_64", family: "mac"
```
```
[INFO]
[INFO] --- maven-source-plugin:3.0.1:jar-no-fork (attach-sources) @ client-java-api ---
[INFO]
[INFO] --- maven-install-plugin:3.0.0-M1:install (default-install) @ client-java-api ---
[INFO] Installing /Users/runzexia/IdeaProjects/kubernetes-client-office/java/kubernetes/target/client-java-api-4.0.0-beta2-SNAPSHOT.jar to /Users/runzexia/.m2/repository/io/kubernetes/client-java-api/4.0.0-beta2-SNAPSHOT/client-java-api-4.0.0-beta2-SNAPSHOT.jar
[INFO] Installing /Users/runzexia/IdeaProjects/kubernetes-client-office/java/kubernetes/pom.xml to /Users/runzexia/.m2/repository/io/kubernetes/client-java-api/4.0.0-beta2-SNAPSHOT/client-java-api-4.0.0-beta2-SNAPSHOT.pom
[INFO] Installing /Users/runzexia/IdeaProjects/kubernetes-client-office/java/kubernetes/target/client-java-api-4.0.0-beta2-SNAPSHOT-tests.jar to /Users/runzexia/.m2/repository/io/kubernetes/client-java-api/4.0.0-beta2-SNAPSHOT/client-java-api-4.0.0-beta2-SNAPSHOT-tests.jar
[INFO] Installing /Users/runzexia/IdeaProjects/kubernetes-client-office/java/kubernetes/target/client-java-api-4.0.0-beta2-SNAPSHOT-javadoc.jar to /Users/runzexia/.m2/repository/io/kubernetes/client-java-api/4.0.0-beta2-SNAPSHOT/client-java-api-4.0.0-beta2-SNAPSHOT-javadoc.jar
[INFO] Installing /Users/runzexia/IdeaProjects/kubernetes-client-office/java/kubernetes/target/client-java-api-4.0.0-beta2-SNAPSHOT-sources.jar to /Users/runzexia/.m2/repository/io/kubernetes/client-java-api/4.0.0-beta2-SNAPSHOT/client-java-api-4.0.0-beta2-SNAPSHOT-sources.jar
[INFO]
[INFO] --- maven-bundle-plugin:3.5.0:install (default-install) @ client-java-api ---
[INFO] Installing io/kubernetes/client-java-api/4.0.0-beta2-SNAPSHOT/client-java-api-4.0.0-beta2-SNAPSHOT.jar
[INFO] Writing OBR metadata
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:36 min
[INFO] Finished at: 2019-01-29T20:57:21+08:00
[INFO] ------------------------------------------------------------------------
[INFO] Scanning for projects...
[INFO] Inspecting build with total of 1 modules...
[INFO] Installing Nexus Staging features:
[INFO]   ... total of 1 executions of maven-deploy-plugin replaced with nexus-staging-maven-plugin
[INFO]
[INFO] -----------------< io.kubernetes:client-java-examples >-----------------
[INFO] Building client-java-examples 4.0.0-beta2-SNAPSHOT
[INFO] -------------------------------[ bundle ]-------------------------------
[WARNING] The POM for io.kubernetes:client-java:jar:4.0.0-beta2-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for io.kubernetes:client-java-proto:jar:4.0.0-beta2-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.320 s
[INFO] Finished at: 2019-01-29T20:57:24+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project client-java-examples: Could not resolve dependencies for project io.kubernetes:client-java-examples:bundle:4.0.0-beta2-SNAPSHOT: Failed to collect dependencies at io.kubernetes:client-java-api:jar:4.0.0-beta2-SNAPSHOT: Failed to read artifact descriptor for io.kubernetes:client-java-api:jar:4.0.0-beta2-SNAPSHOT: Could not find artifact io.kubernetes:client-java-parent:pom:4.0.0-beta2-SNAPSHOT -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

--- a/examples/createPod.sh
+++ b/examples/createPod.sh
@@ -1,11 +1,26 @@
-#creates a pod and runs Example.java(list pods for all namespaces) on starting of pod
-export REPO_ROOT=$(pwd)/../
+#!/bin/bash
 
-cd ${REPO_ROOT}/kubernetes
+# creates a pod and runs 
+# Example.java(list pods for all namespaces) on starting of pod
+
+# Exit on any error.
+set -e
+
+if ! which minikube > /dev/null; then
+  echo "This script requires minikube installed."
+  exit 100
+fi
+
+dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+export REPO_ROOT=${dir}/..
+
+cd ${REPO_ROOT}
 mvn install
 
 cd ${REPO_ROOT}/examples
 mvn package
+
 eval $(minikube docker-env)
 docker build -t test/examples:1.0 .
 kubectl apply -f test.yaml

